hadoop-yarn-project/hadoop-yarn/bin/FederationStateStore/MySQL/FederationStateStoreTables.sql

```diff
@@ -36,7 +36,7 @@ CREATE TABLE membership(
    state varchar(32) NOT NULL,
    lastStartTime bigint NULL,
    capability varchar(6000),
-   CONSTRAINT pk_subClusterId PRIMARY KEY (subClusterId)
+   CONSTRAINT pk_subClusterId PRIMARY KEY (subClusterId),
```

Comment:
Is this validated by the unit test?

Reply:
Thank you very much for your help in reviewing the code. This part of the modification has been verified. I used the compatibility mode of HSQLDB's Mysql and SqlServer to verify the table and stored procedure scripts respectively.

Reply:
I have tested locally and I found that HSQLDB's compatibility mode for Mysql and SqlServer is not fully compatible with all syntaxes. The verification of this part of the stored procedure may only be verified by providing a test report. I will execute the SQL statement in the database and provide the test report.